Composer/packages/client/src/components/Header.tsx

```diff
@@ -50,17 +52,22 @@ const divider = css`
   margin: 0px 0px 0px 20px;
 `;
 
-const updateAvailableIcon = {
+const controls = css`
+  display: flex;
+  position: absolute;
```

Comment:
nit: this can be achieved without absolute positioning and has the advantage of vertically centering it via css instead of calculations. Please consider that.